x-pack/plugins/cases/server/client/factory.ts

```diff
@@ -97,21 +83,22 @@ export class CasesClientFactory {
       auditLogger: new AuthorizationAuditLogger(auditLogger),
     });
 
-    const userInfo = this.options.caseService.getUser({ request });
+    const caseService = new CaseService(this.logger, this.options?.securityPluginStart?.authc);
+    const userInfo = caseService.getUser({ request });
 
     return createCasesClient({
-      alertsService: this.options.alertsService,
+      alertsService: new AlertService(),
```

Comment:
I was able to pull all the services into the factory here since the HTTP routes no longer need to access them directly.